root/components/Aliases/index.js

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Hrm, in https://github.com/metabrainz/musicbrainz-server/pull/603 we put the index page for the isrc page alongside index.tt, at root/isrc/Index.js. In either case, I'd like the name of the file and the name of the exported component to be the same. What do you think?

If we go with your layout I think I would lowercase 'aliases' in the path and capitalize 'Index.js'.

Reply:
Actually, I found various usages online of this way of organizing components when the components are made of various files. For eg - “How to better organize your React applications?” @alexmngn https://medium.com/@alexmngn/how-to-better-organize-your-react-applications-2fd3ea1920f1

This allows you to reference the Aliases folder as a component importing Aliases imports from the index file. What we can do however is move the component from index.js to its own file and simply import export from index.js. This allows us to keep the react embed names consistent. 'components/Aliases' as opposed to lowercase version of the same.

Reply:
Hmm, makes sense for "scenes" (I don't like that word, would much prefer "entry" or "pages") and components made up of various files. But I think I would not create a Component/index.js structure (as opposed to just Component.js) in cases where it's clearly a leaf component that has no dependencies.

I also generally don't like how quickly the folders become deeply with these very long paths for just a few  components. For example, according to that article your directory structure should be this, I think:
```
root/components/Aliases/index.js
root/components/Aliases/components/AliasTable/index.js
root/components/Aliases/components/AliasTable/components/AliasTableBody/index.js
root/components/Aliases/components/AliasTable/components/AliasTableBody/index.js
```

Reply:
I can see the logic behind it though. One problem I have now with the webpack conversion is finding a way to bundle every entry point (scene/page) for the server without listing them all explicitly. If we used a simplified structure like this:

```
root/pages/Artist/index.js
root/pages/Artist/SomeComponentWhichIsntAPage.js
root/pages/Artist/someUtil.js
root/pages/Artist/Aliases/index.js
```

then it's not clear how to determine which files are the actual pages, and which are the supporting components. If we followed the article more closely, it would be clear (pages would always appear directly under a folder named pages/). What do you think?

Reply:
What's the final verdict on this then?

Reply:
I don't know, but let's keep what you have for now.